tests/dp_computations_test.py

```diff
@@ -172,6 +172,68 @@ def test_compute_dp_sum(self):
         self._test_gaussian_noise(results, 10, params.eps, params.delta,
                                   l2_sensitivity)
 
+    def test_equally_split_budget(self):
+        # The number of mechanisms must be bigger than 0.
+        with self.assertRaises(ValueError):
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 0)
+
+        # Only one mechanism.
+        self.assertEqual(
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 1),
+            [(0.5, 1e-10)])
+
+        # Multiple mechanisms.
+        expected_budgets = [(0.5 / 5, 1e-10 / 5) for _ in range(4)]
+        expected_budgets.append((0.5 - 4 * (0.5 / 5), 1e-10 - 4 * (1e-10 / 5)))
+
+        self.assertEqual(
+            pipeline_dp.dp_computations.equally_split_budget(0.5, 1e-10, 5),
+            expected_budgets
+        )
+
+    def test_compute_dp_mean(self):
```

Comment:
What's the running time of newly added tests?

Reply:
Both tests take around 35 seconds.

Reply:
Thanks, it's reasonable running time